### DISCUSSION TITLE
Information different between npm official site & API

### DISCUSSION THREAD

**Discussion by k0s15** [npm/feedback#1031] [Registry] [answered]
I am reviewing on using npm API to retrieve libraries information.

But I found that some APIs cannot obtain the license info while it can be obtained from web page

Sample:
No license info on API : https://registry.npmjs.org/colors/0.6.2
Contain license info on web page: https://www.npmjs.com/package/colors/v/0.6.2

Anyone has ideas on this?

**Comment by MylesBorins** [comment, marked as answer]
The landing page for a package shows the most recent information. Version 0.6.2 of colors did not have [a license field in the package.json](https://github.com/Marak/colors.js/blob/v0.6.2/package.json) which is why it is not returned by the API
